spec/models/appeal_spec.rb

```diff
@@ -925,21 +925,31 @@
     end
 
     context ".assigned_judge" do
-      let!(:judge) { create(:user) }
-      let!(:judge2) { create(:user) }
-      let!(:appeal) { create(:appeal) }
-      let!(:task) { create(:ama_judge_assign_task, assigned_to: judge, appeal: appeal, created_at: 1.day.ago) }
+      let(:judge) { create(:user) }
+      let(:judge2) { create(:user) }
+      let(:appeal) { create(:appeal) }
+      let!(:task) do
+        create(:ama_judge_assign_task, :cancelled, assigned_to: judge,
+                                                   appeal: appeal, created_at: 1.day.ago)
+      end
       let!(:task2) { create(:ama_judge_assign_task, assigned_to: judge2, appeal: appeal) }
-
       subject { appeal.assigned_judge }
 
-      it "returns the assigned judge for the most recent non-cancelled JudgeTask" do
-        expect(subject).to eq judge2
+      context "with one cancelled task" do
+        it "returns the assigned judge for the most recent non-cancelled JudgeTask" do
+          expect(subject).to eq judge2
+        end
       end
 
-      it "should know the right assigned judge with a cancelled tasks" do
-        task2.update(status: "cancelled")
-        expect(subject).to eq judge
+      context "with multiple cancelled tasks" do
+        before(:each) do
+          task2.update(status: Constants.TASK_STATUSES.cancelled)
+        end
+        let!(:task3) { create(:ama_judge_assign_task, assigned_to: judge2, appeal: appeal, created_at: 1.day.ago) }
+
+        it "should know the right assigned judge" do
+          expect(subject).to eq judge2
+        end
```

Comment:
`task2` and `task3` have the same `assigned_to`. Should `task3` be assigned to `judge3` and
```suggestion
        it "should know the right assigned judge" do
          expect(subject).to eq judge3
        end
```